### PR TITLE
docs(setup): Resolved conficts with other PRs, plus further refinements

### PR DESCRIPTION
**Pull request by dorbin**

### CODE REVIEW

**Review by lwander — APPROVED** [review]

File: setup/install/providers/index.md [hunk 1]
```diff
@@ -1,38 +1,47 @@
...
-* <a href="/setup/install/providers/kubernetes-v2/">Kubernetes V2 (manifest based)</a>
-* <a href="/setup/install/providers/openstack/">Openstack</a>
-* <a href="/setup/install/providers/oracle/">Oracle</a>
-
-See also [`hal config provider`](/reference/halyard/commands/#hal-config-provider).
+Providers](/concepts/providers/) that it supports. So, for Spinnaker to do
+anything you must enable at least one provider, with one Account added for it.
+
+(In fact, if you use a [Distributed
+deployment](/setup/install/environment/#distributed-installation), you'll have a provider (on
```
Comment (skim1420) [inline setup/install/providers/index.md]: Let's be consistent with the definition of Providers and Accounts above, 3 occasions in this paragraph. Users aren't adding providers, they're adding Accounts.

File: setup/install/providers/index.md [hunk 2]
```diff
@@ -1,38 +1,47 @@
...
+Kubernetes) to run Spinnaker. You can use that one as your only provider, or you
+can create an additional provider for each cloud target.)
```
Comment (skim1420) [inline setup/install/providers/index.md]: We should prefer not re-using the same account for both running spinnaker and deploying user workloads. I would rather that we prescribe not re-using, then pointing out that they have the option to do so.

File: setup/install/providers/index.md [hunk 3]
```diff
@@ -1,38 +1,47 @@
...
-* <a href="/setup/install/providers/dcos/">DC/OS</a>
-* <a href="/setup/install/providers/gce/">Google Compute Engine</a>
-* <a href="/setup/install/providers/kubernetes/">Kubernetes (legacy)</a>
-* <a href="/setup/install/providers/kubernetes-v2/">Kubernetes V2 (manifest based)</a>
-* <a href="/setup/install/providers/openstack/">Openstack</a>
-* <a href="/setup/install/providers/oracle/">Oracle</a>
-
-See also [`hal config provider`](/reference/halyard/commands/#hal-config-provider).
+Providers](/concepts/providers/) that it supports. So, for Spinnaker to do
+anything you must enable at least one provider, with one Account added for it.
```
Comment (skim1420) [inline setup/install/providers/index.md]: I prefer the original phrasing of this line. More accurate.

File: setup/install/providers/index.md (hunk 2 above)
Comment (skim1420) [reply setup/install/providers/index.md]: Sorry, this comment wasn't addressed. Thanks.

**Pull request merged**